smriprep/data/io_spec.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
i'm fine with having it, though we'll have to remember to expand it if/when needed after enhancements 🙃

Reply:
Perhaps the fields of the outputnode should be read from this spec, so that we force ourselves to update it every time we change the interface, and have some other way of coordinating fMRIPrep (and particularly preempt api-breaking changes, as it will self-adapt accordingly?)

Reply:
Let me check if I can quickly do this before merging.